src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ CHANGELOG
  * Deprecate making `cache.app` adapter taggable, use the `cache.app.taggable` adapter instead
  * Enable `json_decode_detailed_errors` in the default serializer context in debug mode by default when `seld/jsonlint` is installed
  * Register `Symfony\Component\Serializer\NameConverter\SnakeCaseToCamelCaseNameConverter` as a service named `serializer.name_converter.snake_case_to_camel_case` if available
+ * Add Decorator component integration
+ * Add controller decoration support
```

Comment:
Is this line necessary? It feels like in it already "bundled" in the line above?

Reply:
In this case the Framework integration involved enabling DI only (autowiring, autoconfiguration, compiler pass, etc.) allowing applications to effortlessly use the feature. The second entry focused on enabling the feature itself within the kernel/controller mechanism.

*We might want similar support for messenger/handlers in another PR*